agir/groups/views/api_views.py

```diff
@@ -423,6 +424,41 @@ def get(self):
         pass
 
 
+# Create or delete MuteMessage
+class GroupMessageSwitchNotificationAPIView(UpdateAPIView):
+    permission_classes = (GroupMessagesPermissions,)
+    queryset = MuteMessage.objects.all()
+
+    def get(self, request, *args, **kwargs):
+        person = self.request.user.person
+        result = False
+        try:
+            muted = MuteMessage.objects.get(person=person, message_id=kwargs["pk"])
+            muted.delete()
+            result = "on"
+
+        except MuteMessage.DoesNotExist:
+            MuteMessage.objects.create(person=person, message_id=kwargs["pk"])
+            result = "off"
+
+        return Response(result)
+
+
+# Get MuteMessage info on message_pk
+class GroupMessageNotificationAPIView(ListAPIView):
+    permission_classes = (GroupMessagesPermissions,)
+    queryset = MuteMessage.objects.all()
+
+    def get(self, request, *args, **kwargs):
+        person = self.request.user.person
+        result = False
+        if MuteMessage.objects.filter(person=person, message_id=kwargs["pk"]).exists():
+            result = "off"
+        else:
+            result = "on"
```

Comment:
Pourquoi "on" et "off" et pas True/False ?
```suggestion
        result = MuteMessage.objects.filter(person=person, message_id=kwargs["pk"]).exists()
```

Reply:
Je trouve `on / off` plus explicite que `True / False` pour déterminer le statut silencieux ou non d'un fil de message.
Avec un booléen on ne sait pas si `True` indique que le message est bien silencieux ou s'il est bien actif ?